presto-docs/src/main/sphinx/release/release-0.157.1-t.rst

```diff
@@ -0,0 +1,182 @@
+``authentication.ldap.group-auth-pattern`` (&(objectClass=person)(sAMAccountName=${USER})    (&(objectClass=inetOrgPerson)(uid=${USER})
+                                           (memberof=cn=group,ou=America,dc=domain,dc=com))  (memberof=cn=group,ou=America,dc=domain,dc=com))
+========================================== ================================================= =================================================
+
+
+SQL
+---
+* Support correlated scalar aggregation subqueries
+* Support correlated scalar subqueries
+* Additional prepared statement syntax: DESCRIBE INPUT and DESCRIBE OUTPUT
+
+
+Data Types
+----------
+* Additional Varchar(x) and Char(x) function implementations
+* Additional Decimal functions implementations
+
+Documentation
+-------------
+* Installation of Presto via Presto Admin
+* Presto Tuning Guide
+
+
+Performance
+-----------
+* Faster Decimal implementation
+* Move certain filters in the WHERE clauses to be executed as part of the INNER JOIN rather than a filter after the join. For example: ``SELECT * FROM t t1 JOIN t t2 ON t1.a = t2.a WHERE (t1.b+t2.b)*5 > 100000000``
+* Code generation for joins with filters
+* Merge non-identical windows (for the same `partition by` and `order by` but different frame)
+* Optimize execution order of Windowing functions to minimize the number of times data is repartitioned. For example:
+
+   ``wfunc1() OVER (PARTITION BY A ORDER BY B)``
+
+   ``wfunc2() OVER (PARTITION BY C ORDER BY D)`` 
+
+   ``wfunc3() OVER (PARTITION BY A ORDER BY B)``
+
+   If the execution order is as written in the query, it results in partitioning the data 3 times.
+   If we rearrange to execute wfunc1 -> wfunc3 -> wfunc2 then data is only partitioned twice.
+* Add support for running regular expression functions using the more efficent re2j-td library by setting the session variable ``regex_library`` to RE2J.  The memory footprint can be adjusted by setting ``re2j_dfa_states_limit``. Additionally, the number of times the re2j library falls back from its DFA algorithm to the NFA algorithm (due to hitting the states limit) before immediately starting with the NFA algorithm can be set with the ``re2j_dfa_retries`` session variable.
+* Optimize DictionaryBlock.copyPositions()
+  
+Bugs Fixed
+----------
+* Fix issue “Hive table is corrupt. It is declared as being bucketed, but the files do not match the bucketing declaration. The number of files in the directory (1) does not match the declared.” by fixing support for Hive bucketed tables. See option hive.multi-file-bucketing.enabled in the Presto Hive connector documentation.
+* Allow empty partitions for clustered hive tables
+* Fix issue “low must be less than or equal to high” that can occur with ORC and Character data.
+* Fix incorrect stream property derivations from GroupIdNode
+* Remove broken %w specifier for MySQL date functions
+* RPM to include installation of the Memory connector.
```

Comment:
Why this is here if we don't support InMemory for customers (https://github.com/Teradata/presto/pull/460#discussion_r91100614)?